src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -10,10 +10,12 @@
         {{workItem.id}}
       </div>
     </div>
-    <div class="form-group">
+    <div class="form-group" [ngClass]="{'has-error': !workItem.fields['system.title']}">
       <label class="col-sm-1 control-label" for="wi-detail-title">Title</label>
       <div class="col-sm-5">
         <input type="text" [(ngModel)]="workItem.fields['system.title']" id="wi-detail-title" placeholder="Title (mandatory)" required class="form-control">
+        <span *ngIf="!workItem.fields['system.title']" class="help-block">Title is mandatory.</span>
```

Comment:
Validation should not be added like this. We should probably have directive for validation containing all different types of validation logic and messages.